### ISSUE TITLE
Javascript

### ISSUE THREAD

**Issue by ESAISS** [ESAISS/Cogitare#2]
There is a problem with Javascript code.

**Comment by Techydeveloper12** [comment]
hey i can resolve the bug  please elaborate @ESAISS

**Comment by ESAISS** [comment]
Good morning,
I can't read the value of the entered text. I am working on correcting this problem.

**Comment by Techydeveloper12** [comment]
Ok assign me this task please, I can fix this bug @ESAISS

**Comment by ESAISS** [comment]
Thank you but the problem has been resolved ! Thank you in any case.

**Comment by lokeshvasnik** [comment]
You reponened do you stil have a problem with the code

**Comment by ESAISS** [comment]
No, thank you. It's just if there was another problem with the Javascript code.
